### ISSUE TITLE
Unfilled bigstar

### ISSUE THREAD

**Issue by arve0** [mathjax/MathJax#1196]
Unfilled bigstar is commonly used notation for correlation:

![skjermbilde 2015-05-26 kl 14 12 35](https://cloud.githubusercontent.com/assets/4810521/7812295/4d725a40-03b1-11e5-8ded-ed105a163bb2.png)

As opposed to filled `\bigstar` which conventionally defines convolution. Is it possible to create such an *unfilled* bigstar with macros? I'm not able to figure out.

I have used [this](http://tex.stackexchange.com/questions/156712/latex-command-for-unfilled-bigstar) for latex, but that relies on a couple of packages.

**Comment by pkra** [comment]
>  Is it possible to create such an unfilled bigstar with macros? I'm not able to figure out.

That depends what you're after. On the one hand, you can uses [MathJax's `\unicode`](http://docs.mathjax.org/en/latest/tex.html#unicode-support) and one of the stars in Unicode to generate some valid input; e.g., `\unicode{x2606}` would give [this](http://www.fileformat.info/info/unicode/char/2606/index.htm. 

On the other hand, this won't necessarily render well as a) AFAIK no font MathJax supports has a glyph for this (so MathJax has guess glyph dimensions) b) this won't be treated as an operator (which I'm guessing a TeX package might do).

(Labeling this "expected behavior" for lack of better label.)

**Comment by arve0** [comment]
Thanks! That worked lovely at [math.stackexchange.com](http://math.stackexchange.com/questions/1299460/why-is-my-phase-correlation-not-equal-to-the-real-correlation). Considering this closed.

**Comment by dpvc** [comment]
If you want it to be treated as a binary operator, use `\mathbin{\unicode{x2606}}`
